### PR TITLE
Aressem/kill mbuild

### PR DESCRIPTION
**Pull request by aressem**

### CODE REVIEW

File: functions.cmake [hunk 1]
```diff
@@ -549,9 +550,24 @@ function(install_java_artifact NAME)
...
 function(install_fat_java_artifact NAME)
     install(FILES "target/${NAME}-jar-with-dependencies.jar" DESTINATION lib/jars/)
 endfunction()
+
+function(install_symlink TARGET LINK)
+    install(CODE "execute_process(COMMAND ln -sf ${CMAKE_INSTALL_PREFIX}/${TARGET} \$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/${LINK})")
+endfunction(install_symlink)
+
+function(add_extra_projects)
+    if(EXTRA_PROJECTS)
```
Comment (geirst) [inline functions.cmake]: Consider adding "VESPA_" prefix to environment variable?

**Review by geirst — APPROVED** [review]
Overall looks good, please consider the comment

File: functions.cmake (hunk 1 above)
Comment (aressem) [reply functions.cmake]: I considered it, but the other EXTRA_ stuff does not have it. Like EXTRA_INCLUDE_DIRECTORY, EXTRA_LINK_DIRECTORY etc.

Comment (geirst) [reply functions.cmake]: I see. LGTM

**Comment by geirst** [comment]
@toregge FYI

**Pull request merged**